queries.sql

```diff
@@ -51,3 +51,17 @@ SELECT *FROM animals;
  SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jennifer Orwell' AND species.name = 'Digimon';
  SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
  SELECT owners.full_name FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY COUNT(animals.id) DESC LIMIT 1;
+
+
+
+--  JOIN TABLES QUERY
+
+SELECT animals.name FROM visits  JOIN vets ON visits.vets_id = vets.id  JOIN animals ON visits.animals_id = animals.id WHERE vets.name = 'William Tatcher' ORDER BY date DESC LIMIT 1;
+SELECT COUNT(date) FROM visits JOIN vets ON vets_id = vets.id WHERE vets.name = 'Stephanie Mendez';
+SELECT vets.name, species.name FROM vets LEFT JOIN specializations ON vets.id = vets_id LEFT JOIN species ON species.id = species_id;
+SELECT animals.name,visits.date FROM visits  JOIN vets ON visits.vets_id = vets.id  JOIN animals ON visits.animals_id = animals.id WHERE vets.name = 'William Tatcher' AND date BETWEEN '2020-04-01' AND '2020-08-30';
+SELECT animals.name FROM visits JOIN animals ON animals_id = animals.id GROUP BY animals.name ORDER BY COUNT(animals.id) DESC LIMIT 1;
+SELECT animals.name,visits.date FROM visits JOIN animals ON animals_id = animals.id JOIN vets ON vets_id = vets.id WHERE vets.name = 'Maisy Smith' ORDER BY date LIMIT 1;
+SELECT animals.name,vets.name,date FROM visits JOIN animals ON animals_id = animals.id JOIN vets ON vets_id = vets.id ORDER BY date DESC LIMIT 1;
+SELECT *FROM visits JOIN animals ON animals_id = animals.id JOIN vets ON vets_id = vets.id JOIN specializations ON vets.id = specializations.vets_id WHERE animals.species_id != specializations.species_id;
```

Comment:
- [ ] In the query at line `66` you should be able to count how many times the visits were as specified in the project requirements instead of returning the entire data, so please make sure to fix this by using the aggregate function `COUNT`.

Reply:
Fixed.